UI/src/app/home/home.component.html

```diff
@@ -3,8 +3,9 @@
 <div>
   <div>
     <div id="selectBox">
-    <select  class="form-control" [(ngModel)]="dataset">
```

Comment:
I can not see where you are updating this.dataset, you removed ngModel.

Reply:
I am not using dataset at all. But anyway.. I will have a second look for sure. It works fine after these changes, so I did it. :) Thanks for pointing it out.

Reply:
You are using it to disable or enable Plot button. this.dataset is always default and because of that Plot button is always disabled